### DISCUSSION TITLE
command line tool?

### DISCUSSION THREAD

**Discussion by soloturn** [AMythicDev/minus#100] [General]
is there a binary using this library, to get a command line tool?

**Comment by AMythicDev** [comment]
Having a command-line tool like`less` or `more` for minus is not a scope for this project right now. I believe it's better if the open source community uses minus to build such tools by themselves (and tailor it to their use-case if they want). One can take inspiration from the `less-rs` example on how to get started.

Anyway, if this discussion gets enough support, I won't hesitate to incorporate a `less`-like binary alongside the `nain` library
